test/bin/ci_phase_iso_build.sh

```diff
@@ -75,13 +75,13 @@ run_image_build() {
 
     # Build the images
     # Image build can be optimized in CI based on the job type
+    # - Always build 'base' and 'presubmit' layers
+    # - Only build 'periodic' layer in periodic jobs
     if [ -v CI_JOB_NAME ] ; then
-        $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -g ./image-blueprints/group1
-        $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -g ./image-blueprints/group2
-        # Groups 3 and 4 contains images exclusively used in periodic CI jobs
+        $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -l ./image-blueprints/layer1-base
+        $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -l ./image-blueprints/layer2-presubmit
         if [[ "${CI_JOB_NAME}" =~ .*periodic.* ]]; then
-            $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -g ./image-blueprints/group3
-            $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -g ./image-blueprints/group4
+            $(dry_run) bash -x ./bin/build_images.sh ${build_opts} -l ./image-blueprints/layer3-periodic
```

Comment:
Correct me if I'm wrong, but seems we'd have the same order we already have?

I thought we'd build `rhel93.image-installer` and `rhel93.toml` along `rhel92.toml` in periodics. Then we could get rid of `layer3-periodic/group2` by moving `rhel93-source.toml` to `l3-periodic/group1`

Reply:
That's an optimization we can apply after the rename. To make it straighforward, I did not want to touch the toml structure.

Reply:
The good thing is that restructuring no longer afftects any scripts.